src/main/java/dukeclasses/Storage.java

```diff
@@ -0,0 +1,150 @@
+package dukeclasses;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+import java.util.ArrayList;
+import java.util.Scanner;
```

Comment:
Nice to see the import statements are grouped as mentioned in the coding standard 😄